### ISSUE TITLE
Unable to load .caffemodel weights

### ISSUE THREAD

**Issue by skw90** [Evolving-AI-Lab/fooling#6]
Hi,

I am a beginner in deep learning and programming. I have trained a version of mnist Lenet according to Caffe instructions. However I am unable to load the .caffemodel weights into your experiment despite ensuring the prototxt layers have the same names (after building and running the debug version, I get ~0.10 confidence for all labels, so I am guessing the weights were randomly initialised without the pretrained .caffemodel weights). I suspect this is due to updated Caffe version I am using.

It is mentioned in the installation guide that:
"The specific version provided is different from the Caffe master branch and it has the
modification that enables feeding OpenCV data from memory to a Caffe model for
evaluation via ImageDataLayer."
May I know whether it is possible to port the modification over to the new Caffe version?

Thanks!

**Comment by skw90** [comment]
I cant seem to train using this git's provided Caffe version as well.

**Comment by anguyen8** [comment]
@kienwei : Yes, I think it is possible. I will try to push a new version that soon.

**Comment by skw90** [comment]
@anguyen8 On a related issue, may I know whether mean file subtraction was used to generate mnist fooling images? I understand from the caffe website that lenet does not include mean file subtraction for classification.

**Comment by anguyen8** [comment]
@kienwei : Yes, if you look at the link here: https://github.com/Evolving-AI-Lab/fooling/tree/master/model/lenet you will find it there. But even without it, I think it wouldn't change the overall result.
